play/android-publisher/src/main/kotlin/com/github/triplet/gradle/androidpublisher/internal/DefaultPlayPublisher.kt

```diff
@@ -231,6 +231,8 @@ internal class DefaultPlayPublisher(
                     println("Uploading $thing: ${(it.progress * 100).roundToInt()}% complete")
                 MediaHttpUploader.UploadState.MEDIA_COMPLETE ->
                     println("${thing.capitalize()} upload complete")
+                MediaHttpUploader.UploadState.NOT_STARTED -> TODO()
+                MediaHttpUploader.UploadState.INITIATION_COMPLETE -> TODO()
```

Comment:
Nope, please do this: `-> {}`

Reply:
Oh gotcha. Yeah, I prefer not printing stuff since this is just supposed to show upload progress and we don't really care about the details (those are already logged if you enable Gradle debug logging).